src/core/Notification/Notification.md

```diff
@@ -18,15 +18,14 @@ import {
     This is a small screen error notification text.
   </Notification>
   <Notification
-    id="notification-1"
+    id="notification-id"
```

Comment:
Had to change this because the styleguidist heading had the id `notification-1` which messed up the example.